atd-etl/cris_import/utils/process_pdfs.py

```diff
@@ -19,42 +19,57 @@
 logger = get_logger()
 
 
-def is_new_cr3_form(page):
-    """Determine if the CR3 is following the older or newer format.
+def get_cr3_version(page, page_width):
+    """Determine the CR3 from version.
 
     The check is conducted by sampling if various pixels are black.
 
+    On August 27, 2024 CRIS started delivering all CR3s using a
+    smaller page size. This function was adapted to handle the
+    legacy large format page and the new smaller page size.
+
     Args:
         page (PIL image): the pdf page as an image
+        page_width (int): the width of the PDF in points
 
     Returns:
-        bool: true if the provided page passes the sampling tests
+        str: 'v1_small', 'v1_large','v2_large', or 'v2_small'
     """
-    new_cr3_form = True
-    for pixel in NEW_CR3_FORM_TEST_PIXELS:
+    page_size = "small" if page_width < 700 else "large"
+    test_pixels = NEW_CR3_FORM_TEST_PIXELS[page_size]
+
+    for pixel in test_pixels:
         rgb_pixel = page.getpixel(pixel)
-        if rgb_pixel[0] != 0 or rgb_pixel[1] != 0 or rgb_pixel[2] != 0:
-            new_cr3_form = False
-            break
-    return new_cr3_form
+        if rgb_pixel[0] > 5 or rgb_pixel[1] > 5 or rgb_pixel[2] > 5:
```

Comment:
i was having trouble hitting pure black pixels in the smaller format, so i added this narrow tolerance of  `0` to `5` in any of the color bands.